src/kudu/consensus/consensus_meta.h

```diff
@@ -186,6 +187,15 @@ class ConsensusMetadata : public RefCountedThreadSafe<ConsensusMetadata> {
     return on_disk_size_.load(std::memory_order_relaxed);
   }
 
+  // Adds all the peer_uuid's in 'removed_peers' to the internal list
+  // (removed_peers_) tracking peers that have been removed from the active
+  // config. 'removed_peers_' can only track 'max_removed_peers' peers. So, the
+  // earliest peers are evicted from the list (if needed)
+  void UpdateRemovedPeers(const std::vector<std::string>& removed_peers);
```

Comment:
do we need a backdoor way of clearing this list from raft-consensus?
or having the ability to remove the UUID from that list.
Do we also need a way of accessing this list, so that we can present it as a log-line or status -var?

Reply:
Yes, this is a good idea. I will work on this.